### DISCUSSION TITLE
Can we run the code server on Windows?

### DISCUSSION THREAD

**Discussion by IamSparky** [coder/code-server#2353] [Q&A]
I was not able to find any commands for installing code server on windows . Just wanna know does code server run on Windows system ? If yes can someome please provide me the installation guide for the same !

**Comment by code-asher** [comment]
We don't currently test on Windows so I'm not sure if it works or not. I
think there are some path issues.

But you should be able to try it by using yarn:

```
yarn global add code-server
```

Or by cloning the repository and building it. Either way, you'll need
some dependencies: https://github.com/Microsoft/vscode/wiki/How-to-Contribute#prerequisites

**Comment by bpmct** [comment]
Here is some info on running it with WSL: https://github.com/cdr/code-server/issues/1868
